packages/core/state/metadata/logics.ts

```diff
@@ -73,17 +70,18 @@ const receiveAnnotationsLogic = createLogic({
         // annotations IF the annotations exist in the data source and we have room
         // to display them
         [
+            "File Name",
             AnnotationName.FILE_NAME,
```

Comment:
File name twice? (I might be misunderstanding what this is doing)

Reply:
Until we switch the FMS MongoDB document structure to mirror the names we expect in the CSV this is here to let users use "File Name" as a column name as opposed to "file_name"